locales/zhCN.lua

```diff
@@ -1215,10 +1215,10 @@ for key, value in pairs({
 	[256] = "通缉!",
 	[259] = "半埋的大桶",
 	[261] = "破损的箱子",	-- Damaged Crate
-	--TODO: [270] = "Unguarded Thunder Ale Barrel",
```

Comment:
Where do you get object names? We are getting them from the Wowhead but if you can provide a different source I can try to add it as well. Unless you manually look them up in the game of course.

For example, this object name is not on Wowhead, it's just `[Unguarded Thunder Ale Barrel]` on the [page](https://cn.wowhead.com/object=270/unguarded-thunder-ale-barrel).

Reply:
https://cn.classic.wowhead.com/object=270/%E6%97%A0%E4%BA%BA%E5%AE%88%E5%8D%AB%E7%9A%84%E9%9B%B7%E9%85%92%E6%A1%B6

Reply:
Okay, this is clever but it might cause some issues. The thing is, retail and classic have different localization in some cases.
Example: object 31, [Old Lion Statue](https://www.wowhead.com/object=31/old-lion-statue). It is [陈旧的石狮子](https://cn.wowhead.com/object=31/%E9%99%88%E6%97%A7%E7%9A%84%E7%9F%B3%E7%8B%AE%E5%AD%90) in retail but [石狮子](https://cn.classic.wowhead.com/object=31/%E7%9F%B3%E7%8B%AE%E5%AD%90) in classic.
So if we just pull classic localization when retail is missing it MIGHT be actually wrong for retail. I suppose it is only slightly different though so it is actually better than English string? But it would stop us from ever acquiring the correct retail string so I'll give it some more thought. Probably will just keep a list of missing retail translations and check them during localization but in the meantime use data from classic/tbc.
Thank you for the idea.

Reply:
I have implemented it in #756